test/tools/ossfuzz/protoToAbiV2.h

```diff
@@ -334,14 +334,27 @@ class ProtoConverter
 		return toHex(maskUnsignedInt(_counter, _numMaskNibbles), HexPrefix::Add);
 	}
 
-	static unsigned getArrayLengthFromFuzz(unsigned _fuzz, unsigned _counter = 0)
+	static unsigned getDynArrayLengthFromFuzz(unsigned _fuzz, unsigned _counter)
 	{
-		return ((_fuzz + _counter) % s_maxArrayLength) + 1;
+		return ((_fuzz + _counter) % s_maxDynamicArrayLength) + 1;
+	}
+
+	static unsigned getStaticArrayLengthFromFuzz(unsigned _fuzz)
+	{
+		return (_fuzz % s_maxStaticArrayLength) + 1;
 	}
 
 	static std::pair<bool, unsigned> arrayDimInfoAsPair(ArrayDimensionInfo const& _x)
 	{
-		return std::make_pair(_x.is_static(), getArrayLengthFromFuzz(_x.length()));
+		return std::make_pair(
+			_x.is_static(),
+			getStaticArrayLengthFromFuzz(_x.length())
+		);
+	}
+
+	static bool addCheck(unsigned _counter)
+	{
+		return (_counter % s_arrayCheckFrequency == 0);
```

Comment:
no braces needed here

Reply:
except maybe around the modular op, not sure about the precedence.

Reply:
thanks, this always confuses me :)